src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/GSuiteSteps.java

```diff
@@ -88,8 +92,12 @@ public void the_employee_is_not_created_in_gsuite(String id) {
 
   @Then("the employee {string} is not updated in gsuite")
   public void the_employee_is_not_updated_in_gsuite(String id) {
+    Collection<GatewayEventDTO> events = gatewayEventMonitor.grabEventsTriggered("SENDING_GSUITE_ACTION_RESPONSE", 10, 3000l);
+    assertEquals(1, events.size());
```

Comment:
why are you passing 10 into grabEventsTriggered, but then asserting there is only 1